scripting/include/srccoop/manager.inc

```diff
@@ -515,45 +459,62 @@ methodmap CoopManager
 	
 }
 
-public Action StartTimerCallback(Handle pTimer)
+// Runs every 1 second.
+// 
+public Action Action_StartTimer(Handle hTimer)
```

Comment:
Lets stick with one convention per callback type. I've been using Timer_Name, Hook_Name.

Reply:
Renamed that function and others to use calling convention `Timer_...`.